modules/scanning-images.adoc

```diff
@@ -130,25 +130,25 @@ Scanner identifies vulnerabilities in images that contain the following Linux di
 | `centos:6`^[1]^, `centos:7`^[1]^, `centos:8`^[1]^
 
 | link:https://www.debian.org/releases/[Debian]
-| `debian:10`, `debian:11`, `debian:12`, `debian:unstable`, `distroless`
+| `debian:10`, `debian:11`, `debian:12`, `debian:unstable`^[2]^, `distroless`
 
 | link:https://www.oracle.com/linux/[Oracle Linux]
-| versions 5-9^[2]^
+| `OracleLinux-R5`^[2]^, `OracleLinux-R6`^[2]^, `OracleLinux-R7`^[2]^, `OracleLinux-R8`^[2]^, `OracleLinux-R9`^[2]^
```

Comment:
This is fine with me, though I figured for consistency, we'd want to display here what customer will see, which is `ol:5`, etc